forge/test/model_demos/high_prio/cnn/pytorch/test_detr.py

```diff
@@ -0,0 +1,69 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+
+# SPDX-License-Identifier: Apache-2.0
+# Detr model having both object detection and segmentation model
+# https://huggingface.co/docs/transformers/en/model_doc/detr
+
+import pytest
+import forge
+import requests
+from torchvision import transforms
+from transformers import (
+    DetrForObjectDetection,
+    DetrImageProcessor,
+    DetrConfig,
+    DetrFeatureExtractor,
+    DetrForSegmentation,
+)
+from PIL import Image
+
+detection_variants = ["facebook/detr-resnet-50"]
+segmentation_variants = ["facebook/detr-resnet-50-panoptic"]
+
+# Load the image
+url = "http://images.cocodataset.org/val2017/000000397133.jpg"
```

Comment:
I think its good practice to load the image within each test function. 
This ensures tests are self-contained.
Also, instead of image, I think we should just generate [random torch tensor](https://github.com/tenstorrent/tt-forge-fe/blob/0e7565e0cb496e79d98de18d41f47ba2ab9871e8/forge/test/mlir/resnet/test_resnet_inference.py#L20) like in the example linked :)